core/vm/actor/actor_encoding.hpp

```diff
@@ -34,7 +34,7 @@ namespace fc::vm::actor {
     auto maybe_params = codec::cbor::decode<T>(params_bytes);
     if (!maybe_params) {
       return outcome::failure(v7 ? VMExitCode::kErrSerialization
-                                 : VMExitCode::kErrSerializationPre7);
+                                 : VMExitCode::kOldErrActorFailure);
```

Comment:
Older name were more specific (ErrSerialization + "before v7")

Reply:
But it is more common error.
I think that it is clear here that kOldErrActorFailure error is used if v7 == false.
I could add a comment if you wish